feature-libs/cart/base/components/added-to-cart-dialog/added-to-cart-dialog.component.html

```diff
@@ -1,4 +1,8 @@
-<div #dialog>
+<div #dialog
+[cxFocus]="focusConfig"
+(esc)="dismissModal('gone')"
+class="AddedToCartDialogComponent"
```

Comment:
1) I think the string input to dismissModal() can be improved to be a bit more descriptive.

2) Is the class `AddedToCartDialogComponent` needed?

Reply:
1. Reason changed to: "ESC pressed"
2.  I think ` AddedToCartDialogComponent ` is needed. It was there originally, and without it the dialog is not correctly laid out.

Reply:
Hey @echianese I tested something and it seems to work well, can you test it?

Remove these two lines from #dialog
`[cxFocus]="focusConfig"
 (esc)="dismissModal('gone')"`

And and these two lines to `<div class="cx-dialog-buttons"` on line 72
```
(esc)="dismissModal('Escape clicked')"
[cxFocus]="{ autofocus: true }"
```